### PR TITLE
Add example for scrollbound animations + lightbox ad

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+    <style amp4ads-boilerplate>body{visibility:hidden}</style>
+    <!-- ## Summary-->
+    <!--
+    Sample AMP Ad using [amp-lightbox](https://www.ampproject.org/docs/reference/components/amp-lightbox),
+    and [amp-animation](https://www.ampproject.org/docs/reference/components/amp-carousel) to create an interactive ad.
+    -->
+    <!-- ## Styling -->
+    <!--
+    This is an advanced example that requires some styling to make it look and function properly.
+
```
Comment (sebastianbenz) [inline src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html]: remove empty line

File: src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+<!-- -->
+<!doctype html>
+<html amp4ads lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://amp-ads.firebaseapp.com/dist/amp-inabox.js"></script>
+    <script async custom-element="amp-animation" src="https://amp-ads.firebaseapp.com/dist/v0/amp-animation-0.1.max.js"></script>
+    <script async custom-element="amp-lightbox" src="https://amp-ads.firebaseapp.com/dist/v0/amp-lightbox-0.1.max.js"></script>
+
+    <meta name="amp4ads-id" content="vendor=adsense,type=creative-id,id=1283474">
```
Comment (alanorozco) [inline src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html]: Shouldn't be there. Removed.

File: src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html (hunk 1 above)
Comment (alanorozco) [inline src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html]: Done.

File: src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+      /* Change position of the expand button for the middle card */
+      .card-gauges .expand-button {
+        left: 90px;
+        top: 50px;
+      }
+      /* Lightbox styling */
+      amp-lightbox {
+        font-family: 'Roboto', sans-serif;
+        font-size: 12px;
+        background: rgba(0, 0, 0, 0.5);
```
Comment (alanorozco) [inline src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html]: Done.

File: src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html [hunk 4]
```diff
@@ -0,0 +1,449 @@
...
+              layout="responsive"
+              alt="Tuned Engine"
+              class="a-carousel-img"></amp-img>
+        </div>
+      </div>
+    </amp-ad-banner>
+    <!-- ## Lightboxes -->
+    <!--
+    The amp-lightbox component defines the child elements that will be displayed in a full-viewport overlay.
+
```
Comment (alanorozco) [inline src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html]: Done.

File: src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html [hunk 5]
```diff
@@ -0,0 +1,449 @@
...
+    We set a different id for each lightbox so they can be opened and closed by actions on other elements.
+
```
Comment (alanorozco) [inline src/amp-ads/40_Experimental_Ads/Scrollbound_Lightbox_Ad.html]: Done.

**Pull request merged**